templates/web/hackney/about/privacy.html

```diff
@@ -0,0 +1,388 @@
+[% INCLUDE 'header.html',
```

Comment:
If the only change from fixmystreet-uk-councils is the basis for processing, then I suggest we factor the templates together somehow.